features/issues/api/use-get-issues.tsx

```diff
@@ -2,32 +2,83 @@ import { useEffect } from "react";
 import { useQuery, useQueryClient } from "@tanstack/react-query";
 import { getIssues } from "@api/issues";
 import type { Page } from "@typings/page.types";
-import type { Issue } from "@api/issues.types";
+import { Issue, IssueLevel, IssueStatus } from "@api/issues.types";
 
 const QUERY_KEY = "issues";
 
-export function getQueryKey(page?: number) {
-  if (page === undefined) {
-    return [QUERY_KEY];
+export function getQueryKey(
+  page?: number,
+  status?: string,
+  level?: string,
+  project?: string,
```

Comment:
It might be simpler to define a `IssueFilter` type somewhere at the API level (I mean in the `/api` folder so you don't have to type these fields all the time.